source/server-selection/server-selection.rst

```diff
@@ -521,6 +527,30 @@ Eligibility MUST be determined from ``tag_sets`` as follows:
 - If the ``tag_sets`` list is not empty and no tag set in the list matches any
   candidate server, no servers are eligible servers.
 
+hedge
+`````
+
+The read preference ``hedge`` parameter is a document that configures how the
+server will perform hedged reads. It consists of the following keys:
+
+- ``enabled``: Enables or disables hedging
+
+To enable hedging with server-side defaults, an empty ``hedge`` document must be
+sent to the server. If no ``hedge`` document is passed or if the ``enabled``
+key is ``false``, the server will not perform hedged reads.
```

Comment:
My understanding from the server scope is that if no hedge document is included in the read preference, then it _will_ perform a hedged read.  Can you double-check this?

Reply:
As per [SERVER-45440](https://jira.mongodb.org/browse/SERVER-45440), mongos will implicitly enable hedging when a read preference of "nearest" is specified, even when no hedge document is included. if the read preference is anything other than "nearest", no hedging will be performed. This is possible with [SERVER-45439](https://jira.mongodb.org/browse/SERVER-45439), which allows specifying a hedge document to explicitly enable hedging for other read preferences (except for "primary"). From my understanding, this also allows specifying a hedge document with `enabled: false` to disable hedging when using a "nearest" read preference.

Both tickets have already landed in 4.3.3. The `delay` option, which staggers hedged reads was removed from the current scope and will not be included in 4.4.

Reply:
In that case, this part of the "or" statement is inaccurate:

>  If no ``hedge`` document is passed or [...], the server will not perform hedged reads.

I don't think we have to specify the behavior when no hedge document is passed.  The server may change this from release to release.

Reply:
This also seems inaccurate:

> To enable hedging with server-side defaults, an empty ``hedge`` document must be
sent to the server.

Reply:
Reworded:

> Hedged reads are automatically enabled in MongoDB 4.4+ when using a ``nearest``
read preference. To explicitly enable hedging, the ``hedge`` document must be
passed. An empty document uses server defaults to control hedging, but the
``enabled`` key may be set to ``true`` or ``false`` to explicitly enable or
disable read hedging.